com.ibm.wala.core/src/main/java/com/ibm/wala/cfg/AbstractCFG.java

```diff
@@ -492,25 +492,19 @@ public void addExceptionalEdge(T src, T dst) {
     }
   }
 
-  /*
-   * @see com.ibm.wala.util.graph.Graph#removeNode(com.ibm.wala.util.graph.Node)
-   */
+  /** @see com.ibm.wala.util.graph.Graph#removeNode(Object) */
```

Comment:
I think `Graph` still has a `removeNodeAndEdges` method? https://github.com/wala/WALA/blob/156d3266a974baabd22314f959604b10cc8eb339/com.ibm.wala.util/src/main/java/com/ibm/wala/util/graph/Graph.java#L29

Reply:
Yes, `Graph::removeNodeAndEdges` still exists, and this method implements that interface method. What of it? The original `@see` tag was not a reference to `Graph::removeNodeAndEdges`. It was a reference to `Graph::removeNode`, except with the wrong parameter type. I have corrected that parameter type, as this seems most consistent with the original intent of the existing `@see` tag.